src/pkg/services/m365/api/drive.go

```diff
@@ -136,6 +136,22 @@ func (c Drives) GetItem(
 	return di, nil
 }
 
+func (c Drives) GetItemContent(
+	ctx context.Context,
+	driveID, itemID string,
+) ([]byte, error) {
+
+	dic, err := c.Stable.
+		Client().
+		Drives().
+		ByDriveId(driveID).
+		Items().
+		ByDriveItemId(itemID).Content().Get(ctx, nil)
+
+	return dic, graph.Wrap(ctx, err, "getting item content").OrNil()
```

Comment:
FYI, we're changing standard behavior to prefer `clues.Wrap` instead of graph.Wrap.
```suggestion
	return dic, clues.Wrap(err, "getting item content").OrNil()
```